subprojects/docs/src/docs/release/notes.md

```diff
@@ -97,6 +113,10 @@ buildCache {
 }
 ```
 
+### Putting annotation processors on the compile classpath or explicit `-processorpath` compiler argument
+
+Putting processors on the compile classpath or using an explicit `-processorpath` compiler argument has been deprecated and will be removed in Gradle 5.0. Annotation processors should be added to the `annotationProcessor` Configuration instead. If you don't want any processing, but your compile classpath contains a processor unintentionally (e.g. as part of some library you use), use the `-proc:none` compiler argument to ignore it.
```

Comment:
:x: Typo `Configuration` -> `configuration`

Reply:
That was intentional, since it's the class name. Do we use lower case in our docs for this?

Reply:
We use backticks if we want to refer to a class name.